server/routes/admin/index.js

```diff
@@ -9,4 +9,20 @@ route.use((req, res, next) => (generalApi.isAdmin(req.user) ? next() : res.statu
 route.use('/task', taskRoute);
 route.use('/test', testRoute);
 
+route.post('/info/:category', (req, res) => {
+  generalApi
+    .getPersonsCategoried(
+      req.params.category,
+      req.query.skip,
+      req.query.top,
+      req.body.find,
```

Comment:
find?
Why is it a body's property? Maybe just req.body?
Also i suppose a name like filterConfig will be much better